models/ht_neuron.h

```diff
@@ -280,7 +300,10 @@ class ht_neuron : public Archiving_Node
       DG_GABA_A,
```

Comment:
I would be in favour of changing ``G_NMDA`` to ``G_T`` but there should at least be a comment like
``G_NMDA, // state variable corresponding to g(t), with g_NMDA = m(V, t) * g(t)``
or something equivalent.